svn/trunk/SlickEngagement_Plugin.php

```diff
@@ -394,62 +394,78 @@ public function removeSemicolons($value)
     public function addSlickPageHeader()
     {
         global $post;
+
         echo "\n";
-        echo '<meta property="slick:wpversion" content="1.2.4" />' . "\n";
+        echo '<meta property="slick:wpversion" content="1.2.5" />' . "\n";
         $siteCode = trim($this->getOption('SiteCode'));
+
         if ($siteCode) {
             $adThriveAbTest = false;
-            $adThriveAbFraction = 0.9;
-            $serverUrl = trim($this->getOption('SlickServerUrl', 'https://app.slickstream.com'));
+            $defaultServerUrl = 'https://app.slickstream.com';
+            $serverUrl = trim($this->getOption('SlickServerUrl', $defaultServerUrl));
             if (substr($serverUrl, 0, 11) === 'adthrive-ab') {
                 $pieces = explode(" ", $serverUrl);
-                $serverUrl = 'https://app.slickstream.com';
+                $serverUrl = $defaultServerUrl;
                 $adThriveAbTest = true;
-                if (count($pieces) > 1) {
-                    $fractionValue = intval($pieces[1]);
-                    if ($fractionValue > 0 && $fractionValue < 100) {
-                        $adThriveAbFraction = $fractionValue / 100;
-                    }
-                }
+                $enabledPercent = (count($pieces) > 1) ? intval($pieces[1]) : 100;
             }
-            echo '<script>' . "\n";
-            echo '"use strict";' . "\n";
+
+            $jsBlock = <<<JSBLOCK
+            window.slickAbTestResult = function(percentEnabled, forceEnable = false, testName = 'On') {
```

Comment:
please update default testName to something other than 'On'.  slkOn90=on and slkOn90=off is very confusing for data analysis.  I'm good with slkstm90, slkemb90, or slkab90, anything other than 'On'